BitmovinAdobeAnalytics/README.md

```diff
@@ -17,7 +17,7 @@ This module allows for the integration of your Adobe Experience Media Analytics
 - To install it, simply add it as a dependency to your project. Add the following pod to your Podfile:
```

Comment:
I can't on line 15, but we now depend on cocoapods >= 1.9

Reply:
Done